cmd/atlas/internal/client/driver.go

```diff
@@ -0,0 +1,109 @@
+package client
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"ariga.io/atlas/sql/mysql"
+	"ariga.io/atlas/sql/postgres"
+	"ariga.io/atlas/sql/schema"
+	"ariga.io/atlas/sql/sqlite"
+)
+
+type (
+	// Driver provides to diff, execute and inspect against a Data Source.
+	Driver interface {
+		schema.Differ
+		schema.Execer
+		schema.Inspector
+	}
+	// AtlasDriver implements the Driver interface using Atlas.
+	AtlasDriver struct {
+		schema.Differ
+		schema.Execer
+		schema.Inspector
+	}
+
+	dbName string
```

Comment:
unless you plan to add methods to this type, using string is perfectly fine. if you accept what I suggested about drivers being registered instead of explicitly imported in compile time, this isn't needed.